Fix travis CI

Does currently fail, see https://travis-ci.org/github/tango-controls/TangoSourceDistribution.

The reason is

```
     [exec] sphinx-build -b singlehtml -d build/doctrees   source build/singlehtml

     [exec] Running Sphinx v1.8.5

     [exec] loading translations [en]... done

     [exec] Registering meta-labels...

     [exec] registering lang 

     [exec] registering audience 

     [exec] ... meta-labels registered.

     [exec] making output directory...

     [exec] Makefile:63: recipe for target 'singlehtml' failed

     [exec] 

     [exec] Theme error:

     [exec] sphinx_rtd_theme is no longer a hard dependency since version 1.4.0. Please install it manually.(pip install sphinx_rtd_theme)

     [exec] make: *** [singlehtml] Error 2
```

Fix that by:
- [ ] Installing sphinx_rtd_theme
- [ ] Install pip packages with a fixed version so that we are not suprised again by new versions and breaking changes

--- a/.travis.yml
+++ b/.travis.yml
@@ -12,7 +12,7 @@ before_script:
       libmysqlclient-dev
       ant
   - sudo apt install -y python python-pip
-  - pip install --user Sphinx~=1.0 ipython~=5.0
+  - pip install -r https://raw.githubusercontent.com/tango-controls/tango-doc/9.3.4/requirements.txt
 script:
   - ant build package
 

